cql3/statements/create_table_statement.cc

```diff
@@ -97,24 +97,26 @@ std::vector<column_definition> create_table_statement::get_columns()
 }
 
 future<shared_ptr<cql_transport::event::schema_change>> create_table_statement::announce_migration(service::storage_proxy& proxy, bool is_local_only) {
-    return make_ready_future<>().then([this, is_local_only, &proxy] {
```

Comment:
Why did you unwrap this? It changes the semantics.
Previously, if announce_new_column_family or get_cf_meta_data threw an exception, it would result in an exceptional future, which would be inspected later in the lambda passed to then_wrapped.
Now it will always be propagated out of announce_migration (and not as an exceptional future; it will be thrown).

Reply:
You're right. Will add make_ready_future<> back. Good catch!

Reply:
Fixed. Thanks.